libs/components/layout/src/lib/modules/action-button/action-button-icon.component.ts

```diff
@@ -25,10 +26,17 @@ export class SkyActionButtonIconComponent {
    * set `iconType` to `filter`. SKY UX supports version 4.7.0 of Font Awesome.
    * For more information about icons in SKY UX, see the
    * [icon component](https://developer.blackbaud.com/skyux/components/icon).
+   * @deprecated use `iconName` instead
```

Comment:
This is _absolutely_ a nitpick but in the other 2 places we use "Use `iconName` instead."

Reply:
Done.